packages/orbit-components/src/Loading/index.tsx

```diff
@@ -91,11 +91,10 @@ const Loading = ({
             "overflow-hidden",
             "box-border",
             type === TYPE_OPTIONS.BUTTON_LOADER &&
-              "absolute start-0 top-0 h-full w-full justify-center",
+              "[&_.orbit-loading-spinner]:h-icon-medium [&_.orbit-loading-spinner]:w-icon-medium absolute start-0 top-0 h-full w-full justify-center",
             type === TYPE_OPTIONS.SEARCH_LOADER && "h-[40px] justify-start",
-            type === TYPE_OPTIONS.INLINE_LOADER
-              ? "inline-flex min-h-[19px] justify-center"
-              : "p-sm flex",
+            type === TYPE_OPTIONS.INLINE_LOADER && "inline-flex min-h-[19px] justify-center",
+            type !== TYPE_OPTIONS.INLINE_LOADER && "flex",
```

Comment:
```suggestion
            type === TYPE_OPTIONS.INLINE_LOADER
              ? "inline-flex min-h-[19px] justify-center",
              : "flex",
```

Why can't it just be?

Reply:
It can be done either this or that, I think in this case it's more readable as when it's separated, without ternary :)